R/main.R

```diff
@@ -33,6 +33,8 @@
 #' @param aType Scenario type: either "Reference" or "Hindcast"
 #' @param aIncludeSubsidies Boolean indicating subsidies should be added to profit
 #' @param aDifferentiateParamByCrop Boolean indicating whether all crops should use the same expectation parameters
+#' @param aUseLatinHypercube Boolean indicating that Latin Hypercube Sampling should be used (if FALSE, sobol sequences will be used)
```

Comment:
Butting in on your PR, sorry. Would a better long term design be `sampling_type = c("latin_hypercube", "sobol")`? I.e. if you wanted to allow for more options down the road?